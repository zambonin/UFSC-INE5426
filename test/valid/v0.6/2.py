exec(open('../../../src/scope_manager.py', 'r').read())


s_context()
if True:
    
    s_context()
    if True:
        s_context()
        i = 0
        while (i < 10):
            
            i = (i + 3)
        r_context()
    r_context()
r_context()
