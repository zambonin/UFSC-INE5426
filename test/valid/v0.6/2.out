int var: i, j
if: true
then:
  int var: i
  if: true
  then:
    for: = i 0, < i 10, = i + i 3
    do:
      int var: j
